README.md

```diff
@@ -943,6 +945,7 @@ Table of Contents
    * [airtable.com](https://airtable.com/) — Looks like a spreadsheet, but it's a relational database, unlimited bases, 1,200 rows/base and 1,000 API requests/month
    * [Astra](https://astra.datastax.com/register) — Cloud Native Cassandra as a Service with 5GB free tier
    * [cloudamqp.com](https://www.cloudamqp.com/) — RabbitMQ as a Service, up to 1M messages/month and 20 connections free
+   * [Directus Cloud](https://directus.io) — Directus as a service. An open-source data platform that automatically generates a dynamic REST and GraphQL API for pure SQL databases.
```

Comment:
Please list it once only in the most appropriate place. Also update it with details of what free resources the SaaS service provide

Reply:
To me this appears only free when self-hosted, whats the free service the SaaS provides?

Reply:
Hey @ripienaar — we've removed the SaaS category (see commit below) and kept only the "CMS" category. Hopefully this clears things up!

Reply:
@benhaynes This is a list for SaaS that has a free tier. To qualify the free service has to be delivered as a SaaS.

Reply:
That makes sense. The same edits still apply, as our entire On-Demand Cloud platform (and therefore our entire product) is available for open-source maintainers. You can see the last FAQ on our pricing page, which describes our support for free instances (for verified OSS maintainers):

> **Do you offer discounts to non-profits or open-source projects?**
> Yes, we do! Being OSS ourselves, we're aware of how important it is to support open-source maintainers. If you are the maintainer of an open-source project, contact us to get approved for a free On-Demand Cloud Project.

https://directus.io/pricing/

Reply:
OK thanks, in the past I denied entries like this since there's no stated criteria for what would constitute an approvable open source project.

Most services that has policies like this take a github signup login and only accepts public report with perhaps compatible license or something.

As stand thats a bit open ended

Reply:
Makes total sense, and we've certainly had to jump through hoops to get approved with certain vendors.

Our (very simple) approval policy: **Directus Cloud email must match a GitHub account that is the primary maintainer of an Open-Source repository using any compatible OSS license (100% of codebase) and must have existed for at least 6 months and/or have 100 stars.**

We have this documented internally, but we can publish that if required. 🙂

Reply:
If that is in the FAQ entry it’s a no brainier. Merged easily then :)

Reply:
Awesome. We've updated that test to be more specific: https://directus.io/pricing/